src/unix/linux_like/android/b64/riscv64/mod.rs

```diff
@@ -362,5 +362,17 @@ pub const SYS_process_mrelease: ::c_long = 448;
 pub const SYS_futex_waitv: ::c_long = 449;
 pub const SYS_set_mempolicy_home_node: ::c_long = 450;
 
+// From NDK's asm/auxvec.h
+pub const AT_SYSINFO_EHDR: ::c_ulong = 33;
+pub const AT_L1I_CACHESIZE: ::c_ulong = 40;
+pub const AT_L1I_CACHEGEOMETRY: ::c_ulong = 41;
+pub const AT_L1D_CACHESIZE: ::c_ulong = 42;
+pub const AT_L1D_CACHEGEOMETRY: ::c_ulong = 43;
+pub const AT_L2_CACHESIZE: ::c_ulong = 44;
+pub const AT_L2_CACHEGEOMETRY: ::c_ulong = 45;
+pub const AT_L3_CACHESIZE: ::c_ulong = 46;
+pub const AT_L3_CACHEGEOMETRY: ::c_ulong = 47;
+pub const AT_VECTOR_SIZE_ARCH: ::c_ulong = 9;
+
```

Comment:
These should get an API check file, could you start `libc-test/semver/android-riscv64.txt`?

Also same as aarch64, any point of `AT_MINSIGSTKSZ`?

Reply:
Hi, I tried to add an API check for riscv but discovered that CI won't check it because it's tier 3. Is it still reasonable to add this since it may break without noticing because CI won't check?

Reply:
It won't get run in CI, but we may as well make it available for anyone who runs tests locally. And it will give us a starting point if the target is ever promoted.

Reply:
sure. I will try to make one.
btw I used to make it manually, is there any tools for autogen?

Reply:
It's just manual. It would be nice if it also made note of things that aren't listed to make this easier, but I'm not sure how exactly that would work.

Reply:
Done